docs/SETUP.md

```diff
@@ -56,6 +56,20 @@ CREATE TABLE IF NOT EXISTS ecchronos.on_demand_repair_status (
     PRIMARY KEY(host_id, job_id))
     WITH default_time_to_live = 2592000
     AND gc_grace_seconds = 0;
+    
+    CREATE TABLE IF NOT EXISTS ecchronos.repair_history(
+    table_id uuid, 
+	node_id uuid, 
+	repair_id timeuuid, 
+	job_id uuid, 
```

Comment:
If you could fix the indentation so it's same as for other examples that would be great.

Reply:
![image](https://user-images.githubusercontent.com/121041937/209097998-3eda7bf4-debd-4fc4-b023-5580f6a21974.png)